src/app/backend/events.go

```diff
@@ -72,8 +73,27 @@ func GetEvents(client *client.Client, namespace string) (*Events, error) {
 		Namespace: namespace,
 	}
 
-	for _, element := range list.Items {
-		events.Events = append(events.Events, Event{
+	// Get events for replica set.
+	selector, _ := fields.ParseSelector("involvedObject.name=" + replicaSet.Name)
```

Comment:
You ignore error here. Just add `if err != nil {` boilerplate everywhere. It is easy. With that we'll not forget about this in the future.